h2o-r/h2o-package/R/explain.R

```diff
@@ -1614,21 +1623,20 @@ h2o.varimp_heatmap <- function(object,
   return(p)
 }
 
-#' Model Prediction Correlation Heatmap
+
+#' Model Prediction Correlation
 #'
-#' This plot shows the correlation between the predictions of the models.
+#' Get a data.frame containing the correlation between the predictions of the models.
 #' For classification, frequency of identical predictions is used. By default, models
 #' are ordered by their similarity (as computed by hierarchical clustering).
 #'
-#' @param object An H2OAutoML object or list of H2O models.
-#' @param newdata An H2O Frame.  Predictions from the models will be generated using this frame, 
+#' @param object A list of H2O models, an H2O AutoML instance, or an H2OFrame with a 'model_id' column (e.g. H2OAutoML leaderboard)..
+#' @param newdata An H2O Frame.  Predictions from the models will be generated using this frame,
 #'                so this should be a holdout set.
-#' @param top_n Integer specifying the number models shown in the heatmap (used only with an 
+#' @param top_n Integer specifying the number models shown in the heatmap (used only with an
```

Comment:
isn't it deprecated in R? at least in doc.

Reply:
does it require to define a generic method like with `varimp`?

Reply:
Good catch, thank you!

Reply:
> does it require to define a generic method like with varimp?

I think it's probably better to deprecate it and motivate users to use the more flexible way - the leaderboard.